---
title: Vitae
layout: page
permalink: /cv/index.html
---
## Amoolya Srinivasa
<!---[srinivasa.a@northeastern.edu](mailto:srinivasa.@northeastern.edu) --->
Boston, MA
## Preface
It is possible that my fascination for science began from my first glance through the microscope at Paramecium back in the biology lab at school. Since then I have shown a predilection to learn about the world imperceptible by human eyes. As I got older I was intrigued by the interrelation of biology with computation, mathematics and technology, which led me to be inspired by the vast field of bioinformatics that perfectly amalgamates two of my most passionate fields - computer science and biology. My aim is fueled by a conviction to make a dent, if not a difference, in the field of medical drug designing, machine learning in bioinformatics, stem cell and cancer therapeutics. 

## Education
***Masters — Bioinformatics*, Northeastern University**, Boston, MA. Sep 2021 - May 2023<br>
***Bachelors — Biotechnology*, Dayananda Sagar College of Engineering**, Bangalore, India. Sep 2017 - Aug 2021<br>

## Experience
**NextRNA Therapeutics, Boston, MA, Jan 2023 - Present**<br>
***Computational Biology - Data Science Co-Op***, *[NextRNA Therapeutics](http://nextrna.com/)*<br>

* Executing advanced computational modeling techniques to drive target identification and drug discovery initiatives
* Employing cutting-edge machine learning methodologies to develop complex networks for comprehensive gene expression analysis and facilitating patient stratification
* Devising computational tools to design comprehensive analyses of RNA and protein secondary and tertiary structures, with a focus on facilitating the development of lncRNA-directed therapeutics

**Dong Theoretical and Computational Chemistry Lab, Boston, MA, Jul 2022 - Dec 2022**<br>
***Masters Graduate Research Assistant***, *[The Dong Lab](https://sijiadong.com/)*<br>

* Collaborated with the team on the design of a NoSQL database for enzymes with photoactivatable biocofactors
* Utilized advanced Python programming skills to design and implement highly efficient scripts for automating the analysis of chemical names, reactions, solvents, and other small molecule nomenclatures from abstracts of research papers
* Identified natural language processing techniques to extract and curate all possible combinations of enzymes and their corresponding substrates in the database, streamlining the process of data curation and analysis

**Northestern University, Boston, MA, Jan 2022 - Dec 2022** <br>
***Teaching Assistant***, *[Northeastern University, Boston](https://cos.northeastern.edu/master-of-science-in-bioinformatics/)*
* Assessed, mentored and graded python scripts of over 80 Bioinformatics grad students for various Bioinformatics tasks and provided effective, timely feedback to help learn Python programming for Bioinformatics
* Mentored and provided guidance to students, leveraging a deep understanding of their unique learning styles to facilitate their academic success
* Demonstrated exemplary leadership skills by establishing three weekly office hours to provide academic support and address students' inquiries
* Delivered engaging and informative lectures to the class, utilizing dynamic teaching techniques to promote active participation and foster a collaborative learning environment

**RV College of Engineering, Bangalore, India, Aug 2020 - Dec 2020**<br>
***Research Intern***, *[RVCE](https://www.rvce.edu.in/)* <br>
* Conducted a thorough analysis of strategies to mitigate the undesirable side effects associated with conventional drug therapies
*  Investigated 35 naturally occuring alternative drug molecules as a potential replacement for aspirin and other traditional medications in the treatment of cardiovascular diseases to advance patient outcomes through novel drug discovery research
* Worked with university leadership to implement strategic national and global policies that impact and enhance the academic experience.

**Sri Jayadeva Institute of Cardiovascular Sciences and Research, Bangalore, India, Jul 2019 - Oct 2019**<br>
***Microbiology Intern***, *[Jayadeva hospital](http://jayadevacardiology.com/)*
* Efficiently managed a high-volume workload of 100 patient samples per day at the hospital lab, performing a range of immunological, biochemical, serological and ELISA tests
* Conducted detailed microscopic analyses of urine, nasal, and throat samples to identify the underlying causes of bacterial infections

## Skills
**Programming:** Python, BioPython, R, SQL, UNIX/LINUX shell script, Nextflow<br>
**Machine Learning Algorithms:** Linear/multiple regression, SVM, Neural networks(GNN, CNN, LSTM), Random forest, Logistic regression, knn regression, Naive Bayes, decision trees, RIPPER, Clustering techniques<br>
**Data Analytics:** Binary and multiple classification models, multivariate analysis, time series models, Paraller computing, data visualization<br>
**Bioinformatics:** RNA-Seq data analysis, Molecular dynamics simulation, Trimmomatic, Trinity, DESeq2, ClustalW, edgeR, GATK, BEAST, metagenomeSeq, PLINK, Sequence alignment, Transcriptome assembly, Phylogenetic analysis, Pipeline development, protein/nucleotide secondary and tertiary structure prediction and validation<br>

## Publication
Description of Paenibacillus yunnanensis sp. nov., Isolated from a Tepid Spring: Narsing Rao, M. P., Dong, Z. Y., **Amoolya, S.**, Neelavar, S., Liu, B. B., Guo, S. X., Hozzein, W., & Li, W. J. *Current Microbiology*. **2020**; [https://doi.org/10.1007/s00284-020-02087-z](https://doi.org/10.1007/s00284-020-02087-z)<br>

## Certification courses
**R Programming: Advanced Analytics In R For Data Science on Udemy, Jun June 2021**<br> 
This course provided me with a strong foundation in data preparation techniques in R, which helped me delve deeper into data pre-processing, data manipulation, data visualization and analysis. Throughout the course, some tasks that I learned was how to identify and locate missing records in dataframes, how to apply the Median Imputation and Factual Analysis methods to replace missing records. To help work with missing data, I learned how to use some functions and techniques to minimize loss of data and how to reset the dataframe index. Additionally, I gained experience working with the gsub() and sub() functions for replacing strings. One of the most interesting concepts I learned in the course was the importance of NA as a third type of logical constant to help handle missing data in my datasets. By completing this course, I had a strong fundamental foundation for working with data in R, the multitude of functions to efficiently analyze data and apply advanced techniques to handle all kinds of data as a pre-requisite for machine learning efforts. 

**Whole genome sequencing of bacterial genomes: Offered by Technical UNiversity of Denmark on Coursera, Mar 2020**<br>
In this course, I learned about Whole genome sequencing (WGS) of bacterial genomes, which has become increasingly important in the medical sector. As classical methods were being replaced by WGS technology, bioinformatic tools became essential for analyzing the data and obtaining useful results. By the end of the course, I had a solid understanding of the applications of WGS in surveillance of bacteria, including species identification, typing, and characterization of antimicrobial resistance and virulence traits, as well as plasmid characterization. Throughout the course, I had the opportunity to learn about various online tools and how to use them through demonstrations and exercises. These tools were freely available and enabled me to effectively analyze WGS data.

**The science of stem cells: Offered by American Museum of Natural History on Coursera, Jan 2020**<br>
In this five-part online course, I explored the promise that stem cells hold for the treatment of medical conditions. I learned about the history and basic biology of stem cells, as well as new research techniques. Through the course, I discovered how stem cells could lead to cures for diseases and to individualized medicine. I had the opportunity to hear from Museum scientists, medical researchers at the frontiers of the field, and a panel of bioethics experts who addressed the ethical implications of stem cell research and therapy. I gained a better understanding of what has already been accomplished in the field of stem cell research, what challenges remain, and what medical breakthroughs may lie ahead.

**Genomic Data Science: Offered by Johns Hopkins University on Coursera, Nov 2019**<br>
Through this course, I was able to gain insights about how genomics is revolutionizing medical discoveries and that it's neccessity in understanding the genome and leverage the data and information from genomic datasets. The course covered the concepts and tools to understand, analyze, and interpret data from next-generation sequencing experiments. Through this course, I learned about the most common tools used in genomic data science, including the use of the command line, along with a variety of software implementation tools like Python, R, and Bioconductor. The course was designed to serve as both a standalone introduction to genomic data science or as a perfect complement to a primary degree or postdoc in biology, molecular biology, or genetics. As a scientist in these fields, seeking to gain familiarity in data science and statistical tools to better interact with the data in my everyday work, I found this course to be highly valuable.
