---
title: Vitae
layout: page
permalink: /cv/index.html
---
## Libère Jensen Ndacayisaba
<!---[nalibere@gmail.com](mailto:nalibere@gmail.com) --->
Boston, MA
## Preamble
Humankind's most valuable asset and resource are health and time, respectively. Disease afflicts health and depletes time. There is no greater life purpose than the eradication of maladies and the betterment of human health. Humbled by the multiscale and spatiotemporal complexity of disease pathogenesis, I'm inspired by innovative science and technologies to decipher disease biology and make new medicines, and passionately committed to making therapeutic discovery more precise, predictable, and programmable.

<!--
## Highlights:
* Experience: ~9 years of computational biology and bioinformatics, ~6 years of machine learning, ~5 years of drug discovery, ~4 years of single cell multiomics -->

## Education
***Ph.D. — Medical Biophysics*, Keck School of Medicine of USC**, Los Angeles, CA. Aug 2017 - May 2022<br>
***B.Sc. — Biotechnology*, Syracuse University**, Syracuse, NY. 2015<br>
***Associate's degree - Natural Science: Math & Technology emphasis*, Ohlone College**, Fremont, CA. 2014

## Experience
**NextRNA Therapeutics, Boston, MA, May 2022 - Present**<br>
***Computational Biologist II***, *[NextRNA Therapeutics](http://nextrna.com/)*<br>

* Developping machine intelligence capabilities for lncRNA-directed therapeutics.
* Applying advanced data science and computational modeling for target identification and drug discovery efforts.

**USC Michelson Convergent Science Institute in Cancer, Los Angeles, CA, May 2018 - May 2022**<br>
***Ph.D. Graduate Research Assistant***, *[Convergent Science Institute in Cancer (CSI-Cancer)](https://kuhn.usc.edu/)*<br>

* Headed up a research team combining data science and single cell multiomics to map and predict cancer progression.
* Led multiple research projects in machine/deep learning to predict patient outcomes in solid tumors and blood cancers.
* Developped feed-forward neural network models using multimodal data to predict survival in breast and prostate cancers.
* Developed immunoassays for single cell morphoproteogenomics to characterize and subtype cancer cells in the blood and bone marrow.

**Nucleate, Los Angeles CA, August 2021 - May 2022** <br>
***Strategy Lead***, *[Nucleate, Los Angeles](https://www.nucleate.xyz/locations/los-angeles-ca/)*
* Created a long-lasting strategic vision for enabling the creation and growth of early-stage life science companies in Los Angeles.
* Mapped and connected the LA biontrepreneurial ecosystem to empower the formation of trainee-led bioventures.
* Sourced top-caliber biotech ideas and business trainees and facilitating formation of founding teams via Nucleate Activator program.
* Co-led efforts for expanding the innovation fellowship, creation of the NADP program, and research initiatives

**USC Graduate Student Government, Los Angeles, CA May 2020 - May 2021**<br>
***Director of Globalization and Inclusion***, *[USC GSG](https://gsg.usc.edu/)* <br>
* Served on the Executive Board governing a body of over 27,000 graduate and professional students.
* Oversaw all advocacy and strategic initiatives for undocu+ and international graduate students across 22 schools, colleges, and academic units.
* Worked with university leadership to implement strategic national and global policies that impact and enhance the academic experience.

**Novartis Institutes for BioMedical Research (NIBR), Cambridge, MA, Jan 2016 - Jun 2017**<br>
***Postbac Intern, Data Science***, *CBT Data Science Group, Chemical Biology and Therapeutics (CBT)*
* Developed proteochemometric machine learning models for predictive drug discovery, target identification, and polypharmacology.
* Evaluated the model performance and therapeutic predictability within and across protein families of the targetable human proteome.
* Experimentally validated predicted interactions and delineated the polypharmacology of predicted targets and small molecule candidates.

**Novartis Institutes for BioMedical Research (NIBR), Cambridge, MA, Jun 2015 - Aug. 2015**<br>
***Intern, Computational Biology***, *Screening Informatics Group, Developmental and Molecular Pathways (DMP)*
* Developed machine learning models to predict siRNA off-targetting and discover pathway nodes in phenotypic screening.
* Investigated top features of miRNA and siRNA that predict off target effects in RNA interference experiments.
* Demonstrated the utility of prediction models for optimizing experimental design for phenotypic screening in target ID and drug discovery.

**Syracuse University, Syracuse, NY, 2014 - Dec 2015**<br>
***McNair Scholar Undergraduate Researcher***, [Lab of Yan-Yeung Luk](https://biomaterials.syr.edu/research/luk/), *Department of Chemistry*
* Computationally investigated protein targets for novel disaccharide derivatives (DSDs) with optimally active stereochemistry.
* Sampled the protein conformations of identified adhesion proteins and their binding affinity to bioactive DSDs.
* Explored the utility of DSDs as inhibitors of biofilm formation in *Pseudomonas aeruginosa*, as a `in vitro` model of Cystic Fibrosis.s

**University of California San Francisco, San Francisco, CA, Summer 2014**<br>
***Undergraduate Researcher***, *[Lab of Michael Grabe](https://www.grabelab.org/), Cardiovascular Research Institute*<br>
* Homology modeled the structure of human sodium-glucose co-transporters (hSGLTs) based on structure of bacterial homologues.
* Explored structural conformations of hSGLT1 and hSGLT2 models for optimal binding of Phlorizin, a competitive binder of D-glucose.
* Investigated inhibitors of human SGLTs by computational screening libraries of active compounds and molecular docking of candidates.

**New York State Department of Health, Albany, NY. Summer 2013**<br>
***NSF-REU Undergraduate Researcher***, *[Labs of Nilesh Banavali and Hongmin Li](https://www.wadsworth.org/senior-staff/nilesh-banavali), The Wadsworth Center*
* Computationally analyzed the structural and biophysical properties of the Dengue virus (DENV) NS2/NS3 protease complex.
* Performed molecular screening, docking, binding affinity analysis, and identified top 40 candidate inhibitors.
* Cultured and purified the NS2/NS3 complex and performed `in vitro` screening for experimental validation of top DENV protease inhibitors.

**University of California Berkeley, Berkeley, CA. Summer 2012**<br>
***Undergraduate Researcher***, *Environmental Leadership Pathway, [ESPM Department](https://ourenvironment.berkeley.edu/)*
* Performed DNA extraction, amplification, and sequencing of COX1 and 28S, two evolutionarily conserved genes in Tahitian plant bugs.
* Applied bioinformatics with Bayesian inference methods to calculate phylogenetic probabilities of speciation
* Investigated the morphological and genetic speciation of Pseudoloxops (Hemiptera: Miridae) and identified 5 candidate new species.

## Skills
**Machine Learning:** Supervised and unsupervised learning, classification and regression (RF, NB, SVm), deep neural networks<br>
**Data Analytics:** Linear and non-linear models, multivariate analysis, time series models, data visualization<br>
**Programming:** Python, R, SQL, Bash, Visual Studio, Atom, LaTeX<br>
**Experimental Biology**: Single cell multiomics, microscopy, imaging mass cytometry, protein purification, immunofluorescence, molecular biology methods<br>
**Spoken Languages**: English, French, Kirundi, elementary Swahili

## Publications
1. **Ndacayisaba L.J.** et al., Enrichment-Free Single-Cell Detection and Morphogenomic Profiling of Myeloma Patient Samples to Delineate Circulating Rare Plasma Cell Clones. `Current Oncology`. **2022**; 29(5):2954-2972.[doi.org/10.3390/curroncol29050242](https://doi.org/10.3390/curroncol29050242) <br>
2. **Ndacayisaba L.J.**, Mason J, Kuhn P. (2021). Mathematical Oncology to Integrate Multimodal Clinical and Liquid Biopsy Data for the Prediction of Survival.<br> Book Chapter in *Circulating Tumor Cells: Advances in Liquid Biopsy Technologies, Springer, 2nd Ed.* `Being edited by publisher.`<br>
3. **Ndacayisaba L.J.**, et al. (2022). Characterization of BCMA Expression in Circulating Rare Single Cells of Patients with Plasma Cell Neoplasms. `IJMS`. `in revision`
4. **Ndacayisaba L.J.**, et al. Project Dragonfly manuscript `in preparation`(update coming soon)

<!--
4. **Ndacayisaba L.J.**, et al. [Project Concorde manuscript] `in preparation`
6. **Ndacayisaba L.J.**, et al. [Project Osprey manuscript] `in preparation`
7. -->

## Selected Presentations
#### Invited Talks and Lectures
* Summer 2019, 2020, & 2021. PIBBS Summer Bootcamps for incoming 1st year PhD students, Los Angeles, CA <br>
Topics: Programming for Bioscientists. Taught first year PhD students on foundations of computational sciences for bioscientists, with hands-on programming and data visualization in R and Python. ~30 PhD students each summer.
* Jul. 2020 Syracuse University McNair Scholars Symposium Jul 17 2020 Virtual <br>
Title: Single Cell Biology and Math Modeling to Delineate the Spatiotemporal Progression of Myeloma
* Summer 2020. ACE the CURE: Summer Undergraduate Program Jul 2020 Virtual <br>
Topics: Predictive Math Modeling. Taught a one week-long bootcamp on predictive machine learning to 17 undergraduates
* Nov. 2019. IMC User Group Meeting at Cedars Sinai: Accelerating Translational Science Through Mass Cytometry. Los Angeles, CA<br>
Title: Imaging Mass Cytometry in Liquid Biopsy: Proteomic Characterization of Circulating Rare Events in Single Cell Oncology
* Dec 2016 Co-lecturer: Novartis Postbaccalaureate Scholars qBio Bootcamp. Cambridge, MA<br>
Topics: introductory command line Bash scripting, Python programming, data analysis, and visualization (class of 8 post-baccalaureate scholars)
students during a 2-day long quantitative biology bootcamp.
* Jul 2016 Invited Seminar Speaker, Novartis Summer Scholars Weekly Seminar Series. Cambridge, MA <br>
Topic: Science Communication, Chalk Talk. Led an hour long seminar to 22 undergraduate summer scholars on giving an effective chalk talk

#### Conferences & Symposia Presentations
* **Poster**. *New England Science Symposium (NESS)*. Apr 2017 Boston, MA
Proteochemometric Machine Learning Models for Predictive Drug Discovery
* **Poster**. *Annual Winter Q-bio Meeting*. Feb 2017 Kauai, HI
Proteochemometric Machine Learning Models for Predictive Drug Discovery, Target Identification, and Polypharmacology Deconvolution
* **Talk & Poster**. *ISCB Student Council Symposium at ISMB*. Jul 2016 Orlando, FL
Proteochemometric Machine Learning Models for Predictive Drug Discovery, Target Identification, and Polypharmacology Deconvolution
* **Poster**. *New England Science Symposium (NESS)*. Apr 2016 Boston, MA
Exploring miRNA Silencing Mechanisms Using Machine Learning Algorithms to Discover Biologically Relevant Pathway Nodes
* **Poster**. *Annual Biomedical Research Conference for Minority Students (ABRCMS)*. Nov 2015 Seattle, WA
Exploring miRNA Silencing Mechanisms Using Machine Learning Algorithms
* **Talk**. *Leadership Alliance National Symposium (LANS)*. Jul 2015 Stamford, CT
Exploring miRNA Silencing Mechanisms Using Machine Learning Algorithms
* **Talk**. *Ronal E. McNair Scholars Research Symposium*. Apr 2015 Syracuse, NY
Prediction and Characterization of Putative Ligand-receptor Interactions in Pseudomonas Aeruginosa
* **Poster**. *Annual Biomedical Research Conference for Minority Students (ABRCMS)*. Nov 2014 San Antonio, TX
Ligand Discovery from a Crystal Structure and Homology Models of Sodium-glucose Co-transporters
* **Talk & Poster**. *UCSF Summer Research Symposium*. Aug 2014 San Francisco, CA
Ligand Discovery from a Crystal Structure and Homology Models of Sodium-glucose Co-transporters
* **Talk**. *NSF-REU Symposium at The Wadsworth Center*. Aug 2013 Albany, NY
Identification of Orthosteric Dengue Protease Inhibitors using Computer-aided Drug Design
* **Poster**. *UC Berkeley ELP Annual Symposium*. Jul 2012 Berkeley, CA
Molecular Taxonomy: Exploring the Speciation of Tahitian Plant Bugs

## Awards and Honors
* May 2022: **PhD Commencement Speaker**, Keck School of Medicine of USC's 2022 graduation
* May 2022: **Recipient**, Order of Arete, USC
* Jul 2021: **Recipient**, Schlegel Family Endowed Fellowship awarded by USC
* Oct 2019: **Winner** of Alix Ventures Award and [Finalist at Health++ hackaton](https://devpost.com/software/guardian-8wgcz1#updates) by Stanford University
* Oct 2019: **2nd place Award**, PIBBS Sci-5: Annual Research Presentation competition by Keck School of Medicine
* 2017 **PhD GEM Associate Fellow**, The National GEM Consortium and University of Southern California
* 2017 **Univeristy Nominee**, AAAS/Science Program for Excellence in Science, by USC
* Jul 2016: **F1000 Outstanding Poster Award**, ISCB Student Council Symposium at ISMB. 1st out of 30 posters.
* Jun 2015: **Research Presentation Award**, Novartis Slide Slam Competition. 3rd out of 21 flash talks.
* May 2015: **Don C. Sawyer III Community Legacy Award**, to Syracuse University’s top minority graduating senior
* Nov 2013: **Dean’s Scholarship**, Syracuse University merit-based renewable annual Scholarship
* 2008-2013: **Various Scholarship Awards**, by Ohlone Foundation, Osher Initiative for Community College Scholarship, and International Students Office

## Service to the Profession
#### Society Membership
* Nov 2021 - Present: **Member**, American Association for Cancer Research (AACR)
* Jun 2019 – Present: **Member**, Society of Hematology Oncology (SOHO)
* Oct 2015 – Present: **Member**, International Society for Computational Biology (ISCB) & ISCB Student Council
* Jul 2017 – 2019: **Member**, American Association for the Advancement of Science (AAAS)
* Dec 2014 – Mar 2018: **Member**, American Society for Pharmacology and Experimental Therapeutics (ASPET)

#### Leadership & Professional Activities
* Sept 2017 – Present: **President and lecturer**, Code On! Lead the team and teach programming to the Keck School of Medicine of USC community
* Jul 2017 – Present: **PIBBS Ambassador**, Recruitment and interviewing prospective Keck SOM PhD students
* Oct 2016 – Jun 2017: **Advisory Committee Member**, TutoringPlus of Cambridge
* Aug 2014 – Dec 2015: **Founder & President**, TheFirst, Syracuse University’s first generation students organization
* Oct 2012 – Jan 2014 **Teaching Assistant** in Biotechnology & Biochemistry. Learning Alliance for Bioscience (LAB) program at Ohlone College
