---
title: "Exploring the Speciation of Tahitian Plant Bugs"
layout: post
date: 2012-07-10
tag:
- taxonomy
- bayesian inference
- mirids
- molecular evolution
- speciation
projects: true
description: "Molecular Taxonomy"
---

**University of California, Berkeley. Summer 2012**

---

## Background
While humans are equipped with the intelligence to make sophisticated machines and travel into space, insects really rule the world. With an estimated 10 million species and more biomass than humans, insects are the most successful creatures on Earth. While the reason for their success remains an important subject of scientific research, there is still an enormous gap in our understanding of insect taxonomy (the identification and description of species and other taxa).
Insects in the family Miridae are known as plant bugs, and are part of a large group of insects called “true bugs” (suborder Heteroptera) that feed by sucking plant (and sometimes animal) juices through their mouth, a straw-like tube. Plant bugs represent the largest Heteropteran family with over 11,000 described species in about 1,400 genera (Wheeler, 2001). While the oldest Homo sapiens fossils date back to about 195,000 years ago (McDougall et al., 2005), plant bugs have roamed the Earth since the Jurassic period 199-145 million years ago (Wheeler, 2001).
This study focuses on a group of plant bugs in the genus Pseudoloxops, which are diverse, understudied and
endemic to the island of Tahiti, French Polynesia. Tahiti is a 403 mi2 volcanic island located in the Archipelago of the Society Islands in the southern Pacific Ocean, and harbors more endemic (i.e. found nowhere else) insect species than any other island in French Polynesia.
Five species of Pseudoloxops were previously known from Tahiti (Knight, 1937), and were described based on morphological data alone, especially color patterns. Here we take an integrative taxonomic approach by combining morphological, molecular and ecological data as lines of evidence to delimit species (Gebiola et al., 2011). We use molecular genetic data as our primary line of evidence, and follow the genealogical species concept, which groups organisms into species based on their common ancestry (Shaw, 2001). The main goal of this project is to find out how many species of Pseudoloxops there are on the island of Tahiti.

## Methods
We collected 18 adult specimens of Pseudoloxops representing a wide range of morphological variation in 2008, 2009, and 2011 from  the island of Tahiti. We also identified the host plant that each specimen was collected from.
Specimens were given a unique identifier (e.g. Z114) and stored in 95% alcohol in a -80°C freezer to preserve DNA.
We extracted DNA and amplified it using the Polymerase Chain Reaction (PCR) for segments of two genes: 814 base pairs of COI (cytochrome C Oxidase I), a mitochondrial gene; and 637 base pairs of 28S, a nuclear ribosomal gene.
We ran gel electrophoresis on the PCR products to verify DNA amplification.

18 successful PCR products were sent to the UC Berkeley DNA Sequencing Facility for sequencing.
Specimen sequences were aligned in the program Geneious Pro 5.6.2 .
The aligned sequences for each gene were entered into Geneious’ MrBayes plug-in to build phylogenetic trees based on Bayesian inference (Huelsenbeck and Ronquist, 2001).

The two sequence alignments were combined to produce a single 50% majority rules consensus phylogeny for all 18 specimens, plus 1 outgroup specimen (Z16). Following the genealogical species concept, we considered a species to be any well-supported (i.e. posterior probability ≥80%) monophyletic clade that was ≥1% genetically distinct from its sister clade.
After identifying putative species, we examined the color patterns and host-plant associations of the individuals within each species to see if morphology and ecology matched with the molecular data.


<!---
<img src="{{site.url}}/assets/images/hdscaplasmacell/pc_mm.png" style="border:none" width="638" />
--->
<!---
<img src="{{site.url}}/assets/images/hdscaplasmacell/gene_rank_plot8.png" style="border:none" width="638" />
--->
