---
title: "Description of _Paenibacillus yunnanensis_ sp. nov., Isolated from a Tepid Spring"
layout: post
date: 2030-13-06
tag:
- taxonomy
- average nucleotide identity
- novel species
- Gram stain-positive
- SYSU G01003T
projects: true
description: "Proposing novel species of the genus _Paenibacillus_"
---

**Dayananda Sagar College of Engineering, India. Summer 2020**

---

## Background
Traditionally, bacteria that displayed rod-like shapes, could thrive in the presence or absence of oxygen, and were capable of forming spores, were placed under the blanket label of Bacillus. However, with the rise of 16S rRNA gene sequencing, it became apparent that the Bacillus group was more complex than previously assumed, and that Paenibacillus was a distinct genus within this group. Paenibacillus was formally introduced into the scientific nomenclature by Ash and his associates, with Paenibacillus polymyxa selected as the type species. Currently, the genus boasts over 240 different species, isolated from diverse sources such as soil, caves, and paperboard. A sediment sample from a tepid spring in Tengchong, located in southwestern China, yielded a Gram stain-positive, rod-shaped and motile bacterium, dubbed SYSU G01003T.In the course of a research project focused on the bacterial diversity of Tengchong's tepid springs in China, the SYSU G01003T strain was identified as an outlier, displaying low sequence identity with other Paenibacillus members. Thus, the current study aims to determine the correct taxonomic classification for SYSU G01003T.

## Methods
* We collected 18 adult specimens of Pseudoloxops representing a wide range of morphological variation in 2008, 2009, and 2011 from  the island of Tahiti. We also identified the host plant that each specimen was collected from.
* Specimens were given a unique identifier (e.g. Z114) and stored in 95% alcohol in a -80°C freezer to preserve DNA.
* We extracted DNA and amplified it using the Polymerase Chain Reaction (PCR) for segments of two genes: 814 base pairs of COI (cytochrome C Oxidase I), a mitochondrial gene; and 637 base pairs of 28S, a nuclear ribosomal gene.
* We ran gel electrophoresis on the PCR products to verify DNA amplification.
* 18 successful PCR products were sent to the UC Berkeley DNA Sequencing Facility for sequencing.
* Specimen sequences were aligned in the program Geneious Pro 5.6.2 .
* The aligned sequences for each gene were entered into Geneious’ MrBayes plug-in to build phylogenetic trees based on Bayesian inference (Huelsenbeck and Ronquist, 2001).
* The two sequence alignments were combined to produce a single 50% majority rules consensus phylogeny for all 18 specimens, plus 1 outgroup specimen (Z16). Following the genealogical species concept, we considered a species to be any well-supported (i.e. posterior probability ≥80%) monophyletic clade that was ≥1% genetically distinct from its sister clade.
* After identifying putative species, we examined the color patterns and host-plant associations of the individuals within each species to see if morphology and ecology matched with the molecular data.

## Results and Discussion

We were able to identify that this microbe showed growth at temperatures between 28-37°C (with an optimum temperature of 37°C), and in pH ranges from 6.0-8.0 (optimal at pH 7.0). It could tolerate NaCl concentrations of up to 2.5% (w/v), but preferred no added NaCl. The cell wall peptidoglycan was composed of meso-2,6-diaminopimelic acid and the sole respiratory quinone was MK-7. The polar lipids identified were diphosphatidylglycerol, phosphatidylglycerol, phosphatidylethanolamine, and a combination of unidentified aminophospholipids, phospholipids, and polar lipids. The prominent fatty acids detected were C16:0, anteiso-C15:0, and C14:0. The genomic DNA G + C content of the type strain was 54.0 mol%. Comparison of the average nucleotide identity (ANI) values between SYSU G01003T and Paenibacillus azotifigens LMG 29963T did not meet the recommended ANI cutoff level (95-96%) for establishing interspecies identity. Therefore, based on these findings, SYSU G01003T is proposed as a new species within the Paenibacillus genus, designated Paenibacillus yunnanensis sp. nov. The type strain of this newly identified species is SYSU G01003T (=KCTC 43132T = CGMCC 1.17384T).
<img src="{{site.url}}/assets/images/publication/phylo.png" style="border:none" width="900" />

* SYSU G01003T is Gram-stain-positive, rod-shaped, and motile with punctiform, circular, white-cream colonies on TSA and grows best at 37°C and pH 7.0, and tolerates up to 2.5% NaCl.
* Its 16S rRNA gene sequence analysis shows it shares 97.3% identity with Paenibacillus azotifigens, and clusters with members of the genus Paenibacillus. The species is positive for starch and gelatin hydrolysis, but negative for oxidase and nitrate reduction.
* Its major fatty acids are C16:0, anteiso-C15:0, and C14:0, and it contains meso-diaminopimelic acid and MK-7 isoprenoid quinone in the peptidoglycan cell walls.
* Its genome has 99 contigs with a total size of 6,407,706 bp and a G + C content of 54.0 mol%. Additionally, it contains 80 tRNAs and seven rRNA, and has a 98.6% completeness and 2.1% contamination.
* Its ANIb and ANIm values with P. azotifigens LMG 29963T are 75.0/83.5%, below the recommended ANI cut-off level for interspecies identity plus it has more gene clusters and singletons than other Paenibacillus members.
* Based on all the above results, SYSU G01003T represents a novel species of the genus Paenibacillus, for which the name _Paenibacillus yunnanensis_ sp. nov. is proposed.

## Here is the link to the [paper](https://link.springer.com/article/10.1007/s00284-020-02087-z) 

